tikv-client/src/main/java/com/pingcap/tikv/meta/TiIndexInfo.java

```diff
@@ -79,12 +79,18 @@ public TiIndexInfo(
 
   public static TiIndexInfo generateFakePrimaryKeyIndex(TiTableInfo table) {
```

Comment:
Add some comments to explain this method

Reply:
Add some comment